### PR TITLE
Improve execution calendar documentation

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 1]
```diff
@@ -21,6 +21,23 @@ One can see the cron expression "0 23 0 ? * MON,TUE,WED,THU,FRI *", which follow
 <<_job_planner_cron_expression_syntax, Quartz Cron Expression Syntax>> section. The cron expression in this example
  executes 23 minutes past every hour, on working days (Monday to Friday).
 
+==== Inclusion Calendar
+
+The purpose of the inclusion calendar is to use ICS file to launch workflows during events. For each events a pre-defined action will be executed an will define when the workflow will be executed.
```
Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: "during events" <-> from a calendar event   ?

Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: "For each events a pre-defined action will be executed an will define when the workflow will be executed." => Given an event, the user select a predefined action triggering a corresponding workflow.

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 2]
```diff
@@ -21,6 +21,23 @@ One can see the cron expression "0 23 0 ? * MON,TUE,WED,THU,FRI *", which follow
...
+===== Available inclusion actions
+
+[cols="1,1", options="header"]
+|===
+
+|Inclusion action
+|Description
+
+|EXECUTE_AT_START
+|The workflow will be executed at each event start.
```
Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: "at each event start" or when an action is selected by the user?

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 3]
```diff
@@ -29,7 +46,76 @@ If an exclusion calendar is not retrievable it is blocking the <<_glossary_workf
...
+===== Available exclusion actions
+
+[cols="1,1", options="header"]
+|===
+
+|Exclusion action
+|Description
+
+|CANCEL_NEXT_EXECUTION
+|All worklfow execution are canceled during the events.
```
Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: if an event is a calendar event, what does "during the events" mean?

Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: Moreover -> All workflow executions ...

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 4]
```diff
@@ -29,7 +46,76 @@ If an exclusion calendar is not retrievable it is blocking the <<_glossary_workf
...
+
+|POSTPONE_TO_NEXT_WORKING_DAY
+|All workflow execution are postponed during events to the next working day. If no execution is planned during an event then no action is taken.
```
Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: All workflow executions ...

Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: "during events" ?

File: src/docs/user/references/JobPlannerServiceReference.adoc (hunk 2 above)
Comment (MaelAudren) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: The workflow is executed at the scheduled time. The action are not selected by the user at runtime but during the workflow definition.

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 5]
```diff
@@ -6,30 +6,86 @@ The <<_glossary_job_planner,*Job Planner*>> schedules <<_glossary_workflow,*Work
...
+				"rule": { "action": "CANCEL_NEXT_EXECUTION" }
+				}
+			     ]
 }
 ----
+
+
+===== Cron
+
+The aim of the cron expression is to launched the planned workflow according to the cron syntax.
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: is to launch

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 6]
```diff
@@ -6,30 +6,86 @@ The <<_glossary_job_planner,*Job Planner*>> schedules <<_glossary_workflow,*Work
    <<_glossary_generic_information,*Generic Information*>> of a <<_glossary_workflow,*Workflow*>>.
 
 === Execution Calendars Object Syntax
-Below is an example of an <<_glossary_execution_calendars_object,*Execution Calendars Object*>>. It has a cron
-expression attached as "cron" and a name as "name".
-The name must be unique and submission will be rejected if a name clashes with an already existing
-<<_glossary_execution_calendars_object,*Execution Calendars Object*>> inside the <<_glossary_job_planner,*Job Planner*>>.
+Below is an example of an <<_glossary_execution_calendars_object,*Execution Calendars Object*>>. The execution calendar
+object can be described according to several token. The execution calendar must have a unique name and submission will be
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: several tokens

Comment (nebilbenmabrouk) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: the submission or the workflow submission

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 7]
```diff
@@ -6,30 +6,86 @@ The <<_glossary_job_planner,*Job Planner*>> schedules <<_glossary_workflow,*Work
...
+rejected if a name clashes with an already existing <<_glossary_execution_calendars_object,*Execution Calendars Object*>>
+inside the <<_glossary_job_planner,*Job Planner*>>. One of the three following token should describe the execution calendar
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: One of the three following tokens

File: src/docs/user/references/JobPlannerServiceReference.adoc (hunk 1 above)
Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: The purpose of the inclusion calendars section is to specify workflow launching policies during calendar events. For instance, automatically launch a specific workflow/action at event start.

File: src/docs/user/references/JobPlannerServiceReference.adoc (hunk 2 above)
Comment (mbenguig) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: The workflow will be submitted at the event start.

**Comment by MaelAudren** [comment]
@nebilbenmabrouk @mbenguig Could you tell me if it is ok for you now ?

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 8]
```diff
@@ -2,53 +2,115 @@
...
    <<_glossary_generic_information,*Generic Information*>> of a <<_glossary_workflow,*Workflow*>>.
 
-=== Execution Calendars Object Syntax
-Below is an example of an <<_glossary_execution_calendars_object,*Execution Calendars Object*>>. It has a cron
-expression attached as "cron" and a name as "name".
-The name must be unique and submission will be rejected if a name clashes with an already existing
-<<_glossary_execution_calendars_object,*Execution Calendars Object*>> inside the <<_glossary_job_planner,*Job Planner*>>.
+=== Execution Calendars Definition Syntax
+
+Job Planner uses an Execution Calendars Definition to know how job will be planned over the time. As shown on the example below, this definition is composed of 4 fields:
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: how the job will be ..

File: src/docs/user/references/JobPlannerServiceReference.adoc [hunk 9]
```diff
@@ -2,53 +2,115 @@
...
+
+ - a name for the definition
+ - a cron expression to define the recurrence (every morning at 6am, etc.)
+ - a set of inclusions calendars to add specific job executions which cannot be defined by a cron expression (holidays, etc.)
+ - a set of exclusions calendars to exclude specific occurence of job executions defined in cron and inclusion definitions (maintenances operations, holidays, etc.)
```
Comment (nebilbenmabrouk) [inline src/docs/user/references/JobPlannerServiceReference.adoc]: - to exclude specific occur"r"ence"s" of the job executions

**Pull request merged**